### DISCUSSION TITLE
How to list external component published as nuget package in Stride's editor

### DISCUSSION THREAD

**Discussion by itn3000** [stride3d/stride#2165] [Q&A] [answered]
I made custom component and want to publish as nuget package.
But I tested in local, my custom component did not appear in another project's component list which refs as nuget package.
If refs as ProjectReference, custom component appeared in component list.

My test code is https://github.com/itn3000/strideexternalnuget
I tested in 4.2.0.2067

**Comment by manio143** [comment, marked as answer]
Can you see if adding a Module initializer with AssemblyRegistry call like [here](https://github.com/stride3d/stride/blob/master/sources/engine/Stride.Physics/Module.cs) will help?

**Reply by itn3000** [reply]
After adding module initializer, it seems to work as expected. Thank you!

**Reply by itn3000** [reply]
I think it is pitfall for package creator, so it may be better adding  description in [manual for creating nuget package](https://doc.stride3d.net/latest/en/manual/nuget/create-packages.html)
